content/contributing-terms.en.md

```diff
@@ -319,6 +323,74 @@ export async function convertImagesToBase64(document, documentDeclaration) {
 }
 ```
 
+#### Terms
```

Comment:
I don't see the added value of this subtitle.
```suggestion

```

Reply:
This is to add a clear marker of where the source document definition ends.

Reply:
At first I found it valuable as it separated the notion of “source document” from the notion of “terms”. However, reading further I agree with @clementbiron: let's make two sections with “single source documents” and “multiple source documents”.